protos/protos/public/uac/RoleService.proto

```diff
@@ -249,6 +249,14 @@ message DeleteRoleBinding {
     };
 };
 
+message DeleteRoleBindings {
```

Comment:
extract this in separate PR

Reply:
extracted PR: https://github.com/VertaAI/modeldb/pull/490